nano/node/node.cpp

```diff
@@ -1120,25 +1120,28 @@ void nano::node::start ()
 
 void nano::node::stop ()
 {
-	logger.always_log ("Node stopping");
-	block_processor.stop ();
-	if (block_processor_thread.joinable ())
+	if (!stopped.exchange (true))
```

Comment:
I guess same can be used in `void nano::bootstrap_server::stop ()` (to not create PR with 1 line)

Reply:
Sure :)